anndata/_io/specs/__init__.py

```diff
@@ -1,2 +1,13 @@
 from . import methods
-from .registry import write_elem, get_spec, read_elem, _REGISTRY, Reader, Writer, IOSpec
+from .registry import write_elem, get_spec, read_elem, Reader, Writer, IOSpec
+from .registry import _REGISTRY  # noqa: F401
+
+__all__ = [
+    "methods",
+    "write_elem",
+    "get_spec",
+    "read_elem",
+    "Reader",
+    "Writer",
+    "IOSpec",
+]
```

Comment:
Again, private module defining exports

```suggestion
```

Reply:
please look at the file: we’re using it as an internal API module.

are you saying that we should get rid of all of those and directly import everything from where it’s defined?